src/shorthands/JBox.jl

```diff
@@ -0,0 +1,126 @@
+function _JBox(
+    cornerpoint1::Point,
+    cornerpoint2::Point,
+    color::String,
+    action::Symbol,
+    vertices::Bool,
+)
+    sethue(color)
+    verts = box(cornerpoint1, cornerpoint2, action, vertices = vertices)
+    return verts[2]
+end
+function _JBox(points::Array, color::String, action::Symbol, vertices::Bool)
+    sethue(color)
+    verts = box(points, action, vertices = vertices)
+    return verts[2]
+end
+function _JBox(
+    pt::Point,
+    width::Real,
+    height::Real,
+    color::String,
+    action::Symbol,
+    vertices::Bool,
+)
+    sethue(color)
+    box(pt, width, height, action, vertices = vertices)
+    return Point(pt.x - width / 2, pt.y + height / 2)
+end
+function _JBox(
+    pt::Point,
+    width::Real,
+    height::Real,
+    cornerradius::Float64,
+    color::String,
+    action::Symbol,
+)
+    sethue(color)
+    box(pt, width, height, cornerradius, action)
+    return Point(pt.x - width / 2, pt.y + height / 2)
+end
+
+"""
+    JBox(cornerpoint1::Point, cornerpoint2::Point; color="black", action=:stroke, vertices=false)
+
+Create a box (rectangle) between two points and do an action.
+Returns the top left corner point of the box.
+"""
+JBox(
+    cornerpoint1::Point,
+    cornerpoint2::Point;
+    color = "black",
+    action = :stroke,
+    vertices = false,
+) =
+    (
+        args...;
+        cornerpoint1 = cornerpoint1,
+        cornerpoint2 = cornerpoint2,
+        color = color,
+        action = action,
+        vertices = vertices,
+    ) -> _JBox(cornerpoint1, cornerpoint2, color, action, vertices)
+
+"""
+    JBox(points::Array; color="black", action=:stroke, vertices=false)
+
+Create a box/rectangle using the first two points of an array of Points to defined opposite corners.
```

Comment:
Could you add a documentation reference to `Points` here?

Reply:
You mean `Point` ? But that is in Luxor so referencing it is unfortunately not that simple.

Reply:
Ah right - never mind! Resolving this.